reconcile/openshift_rolebindings.py

```diff
@@ -90,6 +93,13 @@ def fetch_desired_state(ri, oc_map):
     roles = gqlapi.query(ROLES_QUERY)['roles']
     users_desired_state = []
     for role in roles:
+        exp_date = datetime.strptime(role['expirationDate'], '%Y-%m-%d').date()
```

Comment:
This code is duplicated from above, can you centralize it in some method? Maybe somewhere in `reconcile.utils`?

Reply:
Updated, went ahead and moved the method into specified path.